### ISSUE TITLE
camp realize fails on CityGo example

### ISSUE THREAD

**Issue by gibello** [STAMP-project/camp#68]
#### Characteristics

-   **Issue Type**: bug
-   **Reproducibility**: always
-   **Severity**: minor
-   **Tool/Service/Component**: CAMP
-   **Execution Environment**: Docker
-   **Reporter**: Pierre-Yves Gibello, pyg@ow2.org

#### Description

When following steps in https://stamp-project.github.io/camp/pages/citygo.html:

After "camp generate -d . --coverage", I try "camp realize" and it fails:

```
$ camp realize -d .
CAMP v0.5.1-git.20d1a7d (MIT)
Copyright (C) 2017 -- 2019 SINTEF Digital

Loaded './camp.yml'.
Loading configurations from './out' ...
Unexpected error:
 - [Errno 2] No such file or directory: './out/config_3/images/dashboard_0/Dockerfile'
   In file: /usr/local/lib/python3.5/dist-packages/camp/realize.py
      251: with open(path, "r+") as resource_file:
   Please report this at 'https://github.com/STAMP-project/camp/issues'.

That's all folks!
```
As the error message suggests I file an issue here... I do ;)

**Comment by fchauvel** [comment]
Thanks for the feedback.

We are working on it. Relate to Issues #77 and #80. Note that the source code of the CityGo Application is not open source and that this case study cannot be actually run.

**Comment by fchauvel** [comment]
Closed as similar to #81
